src/components/business/DeleteIdentity.tsx

```diff
@@ -1,7 +1,8 @@
+import Cookie from "js-cookie";
 import { useRouter } from "next/router";
 import React from "react";
 
-import type { IdentityTypes } from "@lib/@types";
+import { IdentityTypes } from "@lib/@types";
```

Comment:
is there a reason?

Reply:
Line 40 I'm using `IdentityTypes` as a value, I can't import it as a type